tests/manual/manual_unwrap_or.rs

```diff
@@ -1145,3 +1211,39 @@ fn allow_match_with_tuple_option_diagnostics() {
 fn allow_if_let_with_array_macro_result_diagnostics() {
     test_lint_diagnostics!(ALLOW_IF_LET_WITH_ARRAY_MACRO_RESULT, @r"");
 }
+
+#[test]
+fn if_let_with_non_droppable_type_diagnostics() {
+    test_lint_diagnostics!(IF_LET_WITH_NON_DROPPABLE_TYPE, @r"");
+}
+
+#[test]
+fn match_with_non_droppable_type_diagnostics() {
+    test_lint_diagnostics!(MATCH_WITH_NON_DROPPABLE_TYPE, @r"");
+}
+
+#[test]
+fn if_let_derive_drop_type_diagnostics() {
+    test_lint_diagnostics!(IF_LET_WITH_DERIVE_DROP_TYPE, @r"
+    Plugin diagnostic: Manual `unwrap_or` detected. Consider using `unwrap_or()` instead.
+     --> lib.cairo:10:5-14:5
+          if let Option::Some(v) = a {
+     _____^
+    | ...
+    |     };
+    |_____^
+    ");
+}
+
+#[test]
+fn match_with_derive_drop_type_diagnostics() {
+    test_lint_diagnostics!(MATCH_WITH_DERIVE_DROP_TYPE, @r"
+    Plugin diagnostic: Manual `unwrap_or` detected. Consider using `unwrap_or()` instead.
+     --> lib.cairo:10:5-13:5
+          match a {
+     _____^
+    | ...
+    |     };
+    |_____^
+    ");
+}
```

Comment:
Please include fixer tests for those 2, as the `manual_unwrap_or` has a fixer itself.